src/content/deadlines/05_thank_you.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: 2024 Team
+subtitle: Meet the team(s)!
+url: /thank-you
+image: "./volunteers-2024.jpg"
+---
+
+Meet the amazing volunteers who spent countless hours and buckets of sweat to make EuroPython 2024 happen.
+All of them having volunteered their time and their energy to make this a great experience for all of us.
```

Comment:
```suggestion
All of them have volunteered their time and energy to make this a great experience for all of us.
```

Reply:
https://langeek.co/en/grammar/course/693/perfect-participles . I think the grammar is as intended. But if you don't like it ... . I'm not too hung up on using it. 
The duplicate their acts as a intensifier of who did the job. But yes it's probably not necessary.

Reply:
1. the duplication of "their" reads fine to me. 
2. Ege is right about the grammar. Your sentence doesn't have a verb here. If you want to use it as you intended, you need to do this: `Meet the amazing volunteers who spent countless hours and buckets of sweat to make EuroPython 2024 happen, all of whom...` or this `All of them, having volunteered their time and their energy, made this a great experience for all of us.` (though the latter is less ideal, unless you pile more adjectives to time and energy)